azure-kusto-data/azure/kusto/data/client.py

```diff
@@ -295,7 +297,7 @@ def _execute(
             endpoint,
             headers=request_headers,
             json=json_payload,
```

Comment:
Wait, up until now we sent BOTH json and data? that doesn't seem right

Reply:
no, up until now you had only one of them at the time, now I have both